### PR TITLE
clarify datetime type option deprecations

### PR DESCRIPTION
**Pull request by xabbuh**
tweaks the changes deprecations documented in #10982 based on the clarifications made in #11004

### CODE REVIEW

File: reference/forms/types/datetime.rst [hunk 1]
```diff
@@ -74,8 +74,10 @@ Defines the ``format`` option that will be passed down to the date field.
 See the :ref:`DateType's format option <reference-forms-type-date-format>`
 for more details.
 
-The ``date_format`` option does not have any effect when the form is rendered
-as an HTML5 datetime input.
+.. versionadded:: 4.3
```
Comment (OskarStark) [inline reference/forms/types/datetime.rst]: You should use newly introduced deprecated directive here , same below

File: reference/forms/types/datetime.rst [hunk 2]
```diff
@@ -74,8 +74,10 @@ Defines the ``format`` option that will be passed down to the date field.
...
+
+    Using the ``date_format`` option when the form is rendered as an HTML
```
Comment (OskarStark) [inline reference/forms/types/datetime.rst]: Splitting HTML and 5 by a newline looks weird

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: reference/forms/types/datetime.rst (hunk 1 above)
Comment (xabbuh) [reply reference/forms/types/datetime.rst]: fixed

File: reference/forms/types/datetime.rst [hunk 3]
```diff
@@ -203,7 +208,10 @@ time_widget
 
 Defines the ``widget`` option for the :doc:`TimeType </reference/forms/types/time>`.
 
-The ``time_widget`` option is ignored when the ``widget`` option is set to ``single_text``.
+.. versionadded:: 4.3
```
Comment (OskarStark) [inline reference/forms/types/datetime.rst]:
```suggestion
.. deprecated:: 4.3
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Christian.

**Pull request merged**